randovania/games/am2r/logic_database/GFS Thoth.txt

```diff
@@ -179,7 +179,7 @@ Extra - lightning_level: 0
 Thoth West Entrance
 Extra - map_name: rm_a8a06
 Extra - minimap_data: [{'x': 43, 'y': 7}]
-Extra - lightning_level: 6
+Extra - light_level: 6
```

Comment:
this supposed to be 6?

Reply:
no, was a typo while i was looking at a8a06.
fixed